spec/unit/pdk/generate/module_spec.rb

```diff
@@ -29,11 +29,13 @@
 end
 
 shared_context 'mock template dir' do
-  let(:test_template_dir) { instance_double(PDK::Module::TemplateDir, metadata: {}) }
+  require 'pdk/module/template_dir/base'
```

Comment:
Rather than adding these requires into the test, it would be better to add `autoload`s for the subclasses at the top of `lib/pdk/module/template_dir.rb`

Reply:
Maybe? Wasn't sure it was worth it as it's only in the spec.